src/AnnouncementBubble/NormalAnnouncementBubble.tsx

```diff
@@ -51,6 +53,8 @@ export const NormalAnnouncementBubble: React.FC<Props> = ({
 }: Props) => {
   const deleteMenu = formDeleteMenu(onDelete);
   const replyButton = formReplyButton(onReply, repliesDisabledMsg, replyButtonText);
+  const numTranslatedLanguagesTooltip =
+    numTranslatedLanguages > 0 ? formNumTranslatedLanguagesTooltip(numTranslatedLanguages) : null;
```

Comment:
minor nit: since this is a pure component, can we rename the function to `NumTranslatedLanguagesTooltip` and update this to `<NumTranslatedLanguagesTooltip  />`